pkg/kubeapiserver/server/insecure_handler.go

```diff
@@ -117,7 +117,7 @@ func serveInsecurely(insecureServingInfo *InsecureServingInfo, insecureHandler h
 	}
 	glog.Infof("Serving insecurely on %s", insecureServingInfo.BindAddress)
 	var err error
-	_, err = server.RunServer(insecureServer, insecureServingInfo.BindNetwork, stopCh)
+	_, err = server.RunServer(insecureServer, insecureServingInfo.BindNetwork, stopCh, nil)
```

Comment:
Does this means that we don't care about the requests come to insecure port?

And a request event like this may be lost?
```
kubectl get pods -s http://120.0.0.1:8080
```

Reply:
I think secure port in most case is open. If insecure port is also open, they use the same audit backend.  I can not image when should secure port not open.